_component_design/help-beacon.md

```diff
@@ -18,44 +17,110 @@ helix-ui-javascript: false
+
+{% endcolumn %}
+
+</div>
 
-### When to use
+</section>
 
-Use a beacon in the following scenarios:
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
 
-- Showing a user new products or features that improve existing functionality.
-- Showing new users the location of commonly used features to aid them in getting started.
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-There are some cases where beacons should not be used:
+## Specifications
 
-- Do **not** use multiple beacons at one time.
-- Do **not** use the beacon to alert users of warnings or alerts.
+Use the following specifications when you construct a beacon.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A beacon consists of the following elements:
+
+- **Beacon (required)**: The beacon is the light source that you place next to the user interface element to which you want to attract the user's attention.
+- **Pulse (required)**: The pulse is the animation that attracts the user's attention. The pulse flashes on and off in one-second intervals.
 
 {% endcolumn %}
 
 {% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% figure [caption:"Beacons continually follow this lifecycle, giving the impression of a pulse to draw user attention."] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/components/help/beacon/beacon-hero.svg"/>
+{% figure [caption:"Beacon composition"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/notifications/beacon/beacon-composition.png" width="164"/>
 {% endfigure %}
 
 {% endcolumn %}
 
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Best practice
+### Style
 
-- Place the beacon to the right or left of navigation items to show location and label.
-- Place the beacon next to labels or section headers.
-- Use the beacon in conjunction with the directional [popover]({{site.baseurl}}/components/popovers.html).
-- Do **not** use the beacon in lieu of user experience best practices about affordance.
-- Do **not** use other forms of notification to deliver the context, such as automatic [tooltips]({{site.baseurl}}/components/tooltip.html) or directional [popovers]({{site.baseurl}}/components/popovers.html).
-- Do **not** obscure the labeled item with the beacon.
+Use these specifications when you construct a beacon.
```

Comment:
I'd say "Use the following specifications..." and end with a colon instead of a period.